client/src/components/SupermarketDisplay.js

```diff
@@ -1,20 +1,14 @@
 import React from "react";
 import PropTypes from "prop-types";
 import styled from "@emotion/styled";
+import SupermarketHeader from "./SupermarketHeader";
+import SupermarketProductsInfo from "./SupermarketProductsInfo";
 
 function SupermarketDisplay({ supermarket }) {
   return (
     <Container>
-      <div>
-        <h3>{supermarket.name}</h3>
-      </div>
-      <div>
-        <p>
-          Products found: {supermarket.matchingProductsFound} of{" "}
```

Comment:
Do you need the {" " } to create white space? Otherwise, this piece of code might be obsolete